src/MudBlazor/Components/Tabs/MudDynamicTabs.razor.cs

```diff
@@ -1,72 +1,103 @@
-﻿using Microsoft.AspNetCore.Components;
+﻿// Copyright (c) MudBlazor 2021
+// MudBlazor licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Components;
 
 #nullable enable
 namespace MudBlazor
 {
+    /// <summary>
+    /// A tab component where the tabs are controlled dynamically, similar to browser tabs.
+    /// </summary>
     public partial class MudDynamicTabs : MudTabs
     {
         /// <summary>
-        /// The icon used for the add button
+        /// The icon for the add button.
         /// </summary>
+        /// <remarks>
+        /// Defaults to <see cref="Icons.Material.Filled.Add"/>.
+        /// </remarks>
         [Parameter]
         [Category(CategoryTypes.Tabs.Appearance)]
         public string AddTabIcon { get; set; } = Icons.Material.Filled.Add;
 
         /// <summary>
-        /// The icon used for the close button
+        /// The icon for the close button.
         /// </summary>
+        /// <remarks>
+        /// Defaults to <see cref="Icons.Material.Filled.Close"/>.
+        /// </remarks>
         [Parameter]
         [Category(CategoryTypes.Tabs.Appearance)]
         public string CloseTabIcon { get; set; } = Icons.Material.Filled.Close;
 
         /// <summary>
-        /// The callback, when the add button has been clicked
+        /// Occurs when the "Add" button has been clicked.
         /// </summary>
         [Parameter] public EventCallback AddTab { get; set; }
 
         /// <summary>
-        /// The callback, when the close button has been clicked
+        /// Occurs when a tab has been closed.
         /// </summary>
         [Parameter] public EventCallback<MudTabPanel> CloseTab { get; set; }
 
         /// <summary>
-        /// Classes that are applied to the icon button of the add button
+        /// The CSS classes applied to the "Add" button.
         /// </summary>
+        /// <remarks>
+        /// Defaults to <c>null</c>. Multiple classes must be separated by spaces.
+        /// </remarks>
         [Parameter]
         [Category(CategoryTypes.Tabs.Appearance)]
         public string AddIconClass { get; set; } = string.Empty;
 
         /// <summary>
-        /// Styles that are applied to the icon button of the add button
+        /// The CSS styles applied to the "Add" button.
         /// </summary>
+        /// <remarks>
+        /// Defaults to <c>""</c>.
+        /// </remarks>
         [Parameter]
         [Category(CategoryTypes.Tabs.Appearance)]
         public string AddIconStyle { get; set; } = string.Empty;
 
         /// <summary>
-        /// Classes that are applied to the icon button of the close button
+        /// The CSS classes applied to the "Close" button.
         /// </summary>
+        /// <remarks>
+        /// Defaults to <c>null</c>. Multiple classes must be separated by spaces.
```

Comment:
```suggestion
        /// Defaults to <c>""</c>. Multiple classes must be separated by spaces.
```

to match the other parameters with `string.Empty`